packages/kit/src/runtime/server/page/actions.js

```diff
@@ -69,23 +69,23 @@ export async function handle_action_json_request(event, options, server) {
 			});
 		}
 	} catch (e) {
-		const error = normalize_error(e);
+		const err = normalize_error(e);
```

Comment:
FYI @bluwy , I think there's a Vite bug where it thinks that `error` is the function imported at the top, not the `const` declared here.

Reply:
👍 looks like @gtm-nayan got it fixed at https://github.com/vitejs/vite/pull/11811

Reply:
Oh nice 🎉